config/config.exs

```diff
@@ -5,4 +5,4 @@ use Mix.Config
       {"localhost", 9093},
       {"localhost", 9094},
     ],
-    consumer_group: false
+    consumer_group: "kafka_ex_dev"
```

Comment:
I see what you mean now.  If the consumer group is falsey then the consumer metadata update timer is not started.

This is sort of minor, but IMO `nil` makes more sense as a default.  That way the type is `binary | nil`.